### PR TITLE
Set fields touched before calling submit callback

### PR DESCRIPTION
**Pull request by MartynasZilinskas**

### CODE REVIEW

File: packages/simplr-forms/src/stores/form-store.ts [hunk 1]
```diff
@@ -379,6 +379,26 @@ export class FormStore extends ActionEmitter {
         }
     }
 
+    public SetFieldsTouched(fieldsIds?: string[]): void {
```
Comment (DovydasNavickas) [inline packages/simplr-forms/src/stores/form-store.ts]: SetFieldsTouched -> TouchFields

File: packages/simplr-forms/src/stores/form-store.ts [hunk 2]
```diff
@@ -379,6 +379,26 @@ export class FormStore extends ActionEmitter {
...
+                if (fieldState != null) {
+                    state.Fields = state.Fields.set(fieldId, fieldState.merge({
+                        Touched: true
+                    } as FieldStoreState));
+                }
+            });
+
+            return this.RecalculateDependentFormStatuses(state);
+        });
+    }
```
Comment (DovydasNavickas) [inline packages/simplr-forms/src/stores/form-store.ts]: Emit action.

File: packages/simplr-forms/src/stores/form-store.ts (hunk 1 above)
Comment (MartynasZilinskas) [inline packages/simplr-forms/src/stores/form-store.ts]: Fixed.

File: packages/simplr-forms/src/stores/form-store.ts (hunk 2 above)
Comment (MartynasZilinskas) [inline packages/simplr-forms/src/stores/form-store.ts]: Fixed.

**Comment by MartynasZilinskas** [comment]
Closes #55.

**Pull request merged**